benchmarks-image-service/stats/package.json

```diff
@@ -0,0 +1,120 @@
+  "_inBundle": false,
+  "_integrity": "sha512-AwqDowltaGTofs5rbHuvNIOoivW+hB2QRc+lIExC/rAmKxdD4ds6FH37I7y86G5AcNmHY4m1nk5BhsAOjCQlJw==",
+  "_location": "/@vx/stats",
+  "_phantomChildren": {},
+  "_requested": {
+    "type": "version",
+    "registry": true,
+    "raw": "@vx/stats@0.0.189",
+    "name": "@vx/stats",
+    "escapedName": "@vx%2fstats",
+    "scope": "@vx",
+    "rawSpec": "0.0.189",
+    "saveSpec": null,
+    "fetchSpec": "0.0.189"
+  },
+  "_requiredBy": [
+    "/"
+  ],
+  "_resolved": "https://registry.npmjs.org/@vx/stats/-/stats-0.0.189.tgz",
+  "_spec": "0.0.189",
+  "_where": "/Users/jagnelli/Desktop/_dev/report-benchmarks/benchmarks-image-service",
+  "author": {
+    "name": "@conglei"
+  },
+  "bugs": {
+    "url": "https://github.com/hshoff/vx/issues"
+  },
+  "dependencies": {
+    "@vx/group": "0.0.189",
+    "@vx/scale": "0.0.189",
+    "classnames": "^2.2.5",
+    "d3-shape": "^1.2.0",
+    "prop-types": "^15.5.10"
+  },
+  "description": "vx stats box violin",
+  "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-proposal-do-expressions": "^7.0.0",
+    "@babel/plugin-proposal-export-default-from": "^7.0.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
+    "@babel/plugin-proposal-function-bind": "^7.0.0",
+    "@babel/plugin-proposal-function-sent": "^7.0.0",
+    "@babel/plugin-proposal-json-strings": "^7.0.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.0.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.4.2",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-16": "^1.0.2",
+    "jest": "^21.2.1",
+    "jest-cli": "^21.2.1",
+    "raf": "^3.4.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-test-renderer": "^16.0.0",
+    "rollup": "^0.67.4",
+    "rollup-plugin-babel": "^4.0.1",
+    "rollup-plugin-commonjs": "^9.1.3",
+    "rollup-plugin-node-resolve": "^3.3.0",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-uglify": "^4.0.0"
+  },
+  "files": [
+    "dist",
+    "build"
+  ],
+  "homepage": "https://github.com/hshoff/vx#readme",
+  "keywords": [
+    "vx",
+    "react",
+    "d3",
+    "visualizations",
+    "charts"
+  ],
+  "license": "MIT",
+  "main": "dist/vx-stats.umd.js",
+  "module": "dist/vx-stats.es.js",
```

Comment:
I think only `"dist/vx-stats.es.js"` and/or `"dist/vx-stats.umd.js",` would need to be commited. Possibly just `"dist/vx-stats.es.js"` if we're bundling because webpack will pick that up